README.rst

```diff
@@ -301,6 +301,12 @@ List of supported endpoints
     amadeus.shopping.flight_offers.pricing.post(flights[0])
     amadeus.shopping.flight_offers.pricing.post(flights[0:2], include='credit-card-fees,other-services')
 
+    # Flight Create Orders
+    amadeus.booking.flight_orders.post(flights[0], traveler)
+
+    # Flight Order Management
+    amadeus.booking.flight_order('eJzTd9f3NjIJdzUGAAp%2fAiY=').get()
```

Comment:
We have a problem with this API, the order id comes from flight booking and is temporary in test... So this example will work for a while then will stop working... Should we just explain that it comes from the previous request' id?